content/integrate/redis-data-integration/ingest/reference/debezium/cassandra.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
I don't think we support Cassandra and MongoDB for Ingest (but check with Yaron, of course :-) ). If you can remove these pages automatically then great, but no big problem - I can just delete them from the published docs.